test/OpenTelemetry.Tests/Metrics/MetricExporterTests.cs

```diff
@@ -0,0 +1,97 @@
+// <copyright file="MetricExporterTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using Xunit;
+
+namespace OpenTelemetry.Metrics.Tests
+{
+    public class MetricExporterTests
+    {
+        [Theory]
+        [InlineData(ExportModes.Push)]
+        [InlineData(ExportModes.Pull)]
+        [InlineData(ExportModes.Pull | ExportModes.Push)]
+        public void FlushMetricExporterTest(ExportModes mode)
```

Comment:
opinion: this test is OK 👍  but it would be easier for me to read if it would be split into 3 distinct unit tests without switch/case statement

feel free to resolve if you disagree